QAT/centos-7.4/media/dev/Dockerfile

```
@@ -115,8 +124,9 @@ RUN mkdir -p /home/build/var/www/tmp/client_body && \
 
 
 # Generate a self-certificate for testing purpose
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN mkdir -p /home/build/etc/nginx && \
-    printf "US\nOR\nPortland\nOregon\nData Center Group\nIntel Corporation\n$(hostname)\nnobody@intel.com\n" | /opt/openssl/bin/openssl req -x509 -nodes -days 30 -newkey rsa:4096 -keyout /home/build/etc/nginx/cert.key -out /home/build/etc/nginx/cert.crt && \
+    printf "US\nOR\nPortland\nOregon\nData Center Group\nIntel Corporation\n%s\nnobody@intel.com\n" "$(hostname)" | /opt/openssl/bin/openssl req -x509 -nodes -days 30 -newkey rsa:4096 -keyout /home/build/etc/nginx/cert.key -out /home/build/etc/nginx/cert.crt && \
```

```suggestion
    printf '%s\n' US OR Portland Oregon "Data Center Group" "Intel Corporation" "$(hostname)" "nobody@intel.com" | /opt/openssl/bin/openssl req -x509 -nodes -days 30 -newkey rsa:4096 -keyout /home/build/etc/nginx/cert.key -out /home/build/etc/nginx/cert.crt && \
```

One thing I recommend for readability is to separate out the strings from all of the newlines